Sources/BraintreePayPal/BTPayPalApprovalURLParser.swift

```diff
@@ -37,7 +37,7 @@ struct BTPayPalApprovalURLParser {
     }
     
     init?(body: BTJSON) {
-        if let payPalAppRedirectURL = body["paymentResource"]["paypalAppApprovalUrl"].asURL() {
+        if let payPalAppRedirectURL = body["agreementSetup"]["paypalAppApprovalUrl"].asURL() {
```

Comment:
Yep - we should be able to begin end to end testing in stage on Monday and need to make sure we have everything set up before we hand off a test flight build to the E2E team. Not having these values set as expected would cause errors. I'll send the docs on slack since the links are to internal resources.

Reply:
Thank you!